### PR TITLE
Add call out to promote Timescale Cloud

### PR DESCRIPTION
**Pull request by ramonguiu**
# Description

Help users discover they can get started more easily with Promscale by using Timescale Cloud

# Links

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: promscale/index.md [hunk 1]
```diff
@@ -5,14 +5,30 @@ product: promscale
 ---
 
 # Promscale
-Promscale is the open source observability backend for metrics and traces
-powered by SQL.
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
```
Comment (Loquacity) [inline promscale/index.md:9]:
```suggestion
Jaeger, and OpenTelemetry. It is built on PostgreSQL and TimescaleDB.
```

File: promscale/index.md [hunk 2]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
 
-It is built on top of PostgreSQL and TimescaleDB. Promscale supports Prometheus metrics and is 100% compliant with PromQL and OpenTelemetry traces.
-Its full SQL capabilities enable developers to correlate metrics, traces, and business data to get valuable insights.
-This is not possible when data is siloed in different systems.
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote
```
Comment (Loquacity) [inline promscale/index.md:11]:
```suggestion
Promscale provides robust and 100% PromQL-compliant Prometheus remote
```

File: promscale/index.md [hunk 3]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+storage and as a durable and scalable Jaeger storage backend. Promscale is a
+certified Jaeger storage backend.
```
Comment (Loquacity) [inline promscale/index.md:13]:
```suggestion
storage. Promscale is a certified Jaeger storage backend that is durable and scalable.
```

File: promscale/index.md [hunk 4]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
 
-*   [Quick start Promscale][quick-start] get started with Promscale in no time.
+Unlike other observability backends, it has a simple and easy-to-manage
+architecture with just two components: the Promscale Connector and the
+Promscale Database (PostgreSQL with the TimescaleDB and Promscale extensions).
```
Comment (Loquacity) [inline promscale/index.md:17]:
```suggestion
Promscale database. The Promscale database consists of PostgreSQL with the 
TimescaleDB and Promscale extensions.
```

File: promscale/index.md [hunk 5]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+
+<highlight type="cloud" header="Promscale with Timescale Cloud" button="Get started for free"
+to="https://console.cloud.timescale.com/signup?campaign=promscale&source=ps-docs-home">
+The best way to use Promscale is with Timescale Cloud, our fully hosted and managed
```
Comment (Loquacity) [inline promscale/index.md:21]:
```suggestion
The best way to use Promscale is with Timescale Cloud, a fully hosted and managed
```

File: promscale/index.md [hunk 6]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+database platform. Enjoy all the best features of Promscale without the
+hassle of managing your database—with automatic backups and failover, high
```
Comment (Loquacity) [inline promscale/index.md:23]:
```suggestion
hassle of managing your database. Timescale Cloud provides automatic 
backups and failover, high
```

File: promscale/index.md [hunk 7]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+availability, flexible scaling, security and data compliance, VPC peering, and
+much more. Get cost savings up to 94% compared to managed Prometheus offerings from
```
Comment (Loquacity) [inline promscale/index.md:25]:
```suggestion
much more. Get cost savings of up to 94% compared to managed Prometheus offerings from
```

File: promscale/index.md [hunk 8]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+AWS, GCP and Grafana.
```
Comment (Loquacity) [inline promscale/index.md:26]:
```suggestion
AWS, Google Cloud Platform, and Grafana.
```

File: promscale/index.md [hunk 9]
```diff
@@ -5,14 +5,30 @@ product: promscale
...
+</highlight>
+
+See the [Promscale with Timescale Cloud installation guide][ptc-install] for details.
```
Comment (Loquacity) [inline promscale/index.md:29]:
```suggestion
For more information about using Promscale with Timescale Cloud, see the 
[Promscale installation guide][ptc-install] for details.
```

File: promscale/installation/index.md [hunk 10]
```diff
@@ -15,6 +15,18 @@ You can install Promscale in several different ways. The method you choose
...
 
+<highlight type="cloud" header="Promscale with Timescale Cloud" button="Get started for free"
+to="https://console.cloud.timescale.com/signup?campaign=promscale&source=ps-docs-install">
+The best way to use Promscale is with Timescale Cloud, our fully hosted and managed
+database platform. Enjoy all the best features of Promscale without the
+hassle of managing your database—with automatic backups and failover, high
+availability, flexible scaling, security and data compliance, VPC peering, and
+much more. Get cost savings up to 94% compared to managed Prometheus offerings from
+AWS, GCP and Grafana.
+</highlight>
```
Comment (Loquacity) [inline promscale/installation/index.md:26]: See previous edits ☝️

File: promscale/index.md (hunk 8 above)
Comment (ramonguiu) [reply promscale/index.md:26]: I don't think we need to expand GCP in the same way we don't expand AWS.

Comment (Loquacity) [reply promscale/index.md]: In that case, we'll need to update our linter.

**Review by Loquacity — APPROVED** [review]

**Pull request merged**